developer-docs-site/docs/tutorials/first-move-module.md

```diff
@@ -259,38 +262,51 @@ For Rust:
 </TabItem>
 </Tabs>
 
-- After a few moments it will mention that "Update the module with Alice's address, build, copy to the provided path,
-  and press enter."
-- In the "Move Window" terminal, and for the Move file we had previously looked at:
-  - Copy Alice's address
-  - Compile the modules with Alice's address by `aptos move compile --package-dir . --named-addresses HelloBlockchain=0x{alice_address_here}`. Here, we replace the generic named address `HelloBlockChain='_'` in `hello_blockchain/move.toml` with Alice's Address
-  - Copy `build/Examples/bytecode_modules/Message.mv` to the same folder as this tutorial project code
-- Return to your other terminal window, and press "enter" at the prompt to continue executing the rest of the code
+On Python terminal:
+ python git:(c6841bd6b0) python3 hello_blockchain.py Message.mv
```

Comment:
What is this `python git:(c6841bd6b0)`? Maybe this is part of the command line prompt on your local terminal? Remove this please. The correct command is `python3 hello_blockchain.py Message.mv`